test/get-random-throw.test.js

```diff
@@ -0,0 +1,17 @@
+import getRandomThrow from '../get-random-throw.js';
+
+const test = QUnit.test;
+
```

Comment:
remember, you can't test functions with random elements!  You'll never know what to expect...